### DISCUSSION TITLE
It's possible to set a page to use the Edge Runtime for SSR in v12? How?

### DISCUSSION THREAD

**Discussion by hcentelles** [vercel/next.js#53639] [Help]
### Summary

I'm using v12.3.4

I noticed that the page router version of this doc [Rendering > Edge and Node.js Runtimes](https://nextjs.org/docs/pages/building-your-application/rendering/edge-and-nodejs-runtimes), don't explain how to do it, [the app router version](https://nextjs.org/docs/app/building-your-application/rendering/edge-and-nodejs-runtimes) instruct to set `export const runtime = 'edge'` on the page.

I try that, but in vercel it renders the page with type "Serverless function" instead of "Edge function".

<img width="357" alt="image" src="https://github.com/vercel/next.js/assets/26359/38037468-8734-4387-9802-f7d62fb0f11b">

**Comment by wwwdepot** [comment]
https://vercel.com/docs/concepts/functions/edge-functions
